launcher.js

```diff
@@ -0,0 +1,45 @@
+// A "launcher" to start the (Node version of the) router from the command line.
```

Comment:
OK, the problem here is that `fluid.require` doesn't respect the `main` setting in `package.json`.  In short, our options at the moment are basically:

1.  Use `node launcher.js` and keep `index.js` for requires.
2. Add documentation indicating that this package cannot be required using `fluid.require`.